travertino/src/travertino/__init__.py

```diff
@@ -1,17 +1,25 @@
-try:
-    # Read version from SCM metadata
-    # This will only exist in a development environment
-    from setuptools_scm import get_version
+def _package_version(file, name):
+    try:
+        # Read version from SCM metadata
+        # This will only exist in a development environment
+        from setuptools_scm import get_version
 
-    # Excluded from coverage because a pure test environment (such as the one
-    # used by tox in CI) won't have setuptools_scm
-    __version__ = get_version("../../..", relative_to=__file__)  # pragma: no cover
-except (ModuleNotFoundError, LookupError):
-    # If setuptools_scm isn't in the environment, the call to import will fail.
-    # If it *is* in the environment, but the code isn't a git checkout (e.g.,
-    # it's been pip installed non-editable) the call to get_version() will fail.
-    # If either of these occurs, read version from the installer metadata.
+        # Excluded from coverage because a pure test environment (such as the one
+        # used by tox in CI) won't have setuptools_scm
+        return get_version(root="../../..", relative_to=file)  # pragma: no cover
+    except (
+        ModuleNotFoundError,
+        LookupError,
+    ):  # pragma: no-cover-if-missing-setuptools_scm
+        # If setuptools_scm isn't in the environment, the call to import will fail.
+        # If it *is* in the environment, but the code isn't a git checkout (e.g.,
+        # it's been pip installed non-editable) the call to get_version() will fail.
+        # If either of these occurs, read version from the installer metadata.
+        import importlib.metadata
 
-    from importlib.metadata import version
+        # The Toga package names as defined in pyproject.toml all use dashes.
+        package = "toga-core" if name == "toga" else name.replace("_", "-")
```

Comment:
The reference to `toga` in a `travertino` namespace is weird; It might be better to reduce this to just the underscore munging, and then use `"toga"` explicitly instead of `__name__` for the core usage.
```suggestion
        package = name.replace("_", "-")
```